components/module-components/Contact/SubmitContactForm.tsx

```diff
@@ -0,0 +1,26 @@
+import axios, { AxiosResponse } from 'axios';
+
+export interface ContactFormData {
+  first_name: string;
+  last_name?: string;
+  email: string;
+  subject?: string;
+  message: string;
+  'cf-turnstile-response'?: string;
+}
```

Comment:
as you can see in the documentation: https://api.learnhub.mk/docs/#website-POSTcontact
all body parameters are required, so we must provide them.
this will apply only to `cf-turnstile-response` since the backend team will need to remove last_name and subject, since we are not sending that data.

Reply:
sending this req will fail with 422